Clean up specs for clarity (#26)
chris-wood/draft-arc: draft-yun-privacypass-arc.md

a presentation limit, denoted `presentation_limit`, a presentation nonce
from a token, denoted `nonce`, and the digest of a token challenge, denoted
`challenge_digest`, verifying a Token requires invoking the VerifyPresentation
function from [Section 4.3.3 of ARC] in the following ways:

~~~
request_context = concat(tokenChallenge.issuer_name,
  tokenChallenge.origin_info,
  tokenChallenge.credential_context,
  issuer_key_id)
presentation_context = concat(tokenChallenge.issuer_name,
  tokenChallenge.origin_info,
  tokenChallenge.redemption_context,
  issuer_key_id)
valid = VerifyPresentation(skI, pkI, request_context, presentation_context, nonce, presentation, presentation_limit)
~~~

This function returns True if the CredentialToken is valid, and False otherwise.

> Implementation-specific steps: to prevent double spending, the server should perform a check that the

Reviewer: ```suggestion
Implementation-specific steps: to prevent double spending, the Origin should perform a check that the
```

> tag (presentation.tag) has not previously been seen. It then stores the tag for use in future double

Reviewer: ```suggestion
presentation (`presentation_nonce` and `presentation`) has not previously been seen for a given request and credential context. It then stores the presentation values for use in future double
```

Author: The server should just store the tag for double spending, as the tag is the "weight bearing" field for rate limiting purposes. The other presentation elements may be re-randomized (and therefore look different) even if the rate limit is exceeded (eg if a nonce is reused), so checking the whole presentation is not the right way to enforce rate limiting.

> spending checks. To reduce the overhead of performing double spend checks, the server can store and

Reviewer: ```suggestion
spending checks. To reduce the overhead of performing double spend checks, the Origin can store and
```

> look up the tags corresponding to the associated request_context and presentation_context values.

Reviewer: ```suggestion
look up the presentation values corresponding to the associated request_context and presentation_context values.
```

Author: Same as above.
